lib/engine/share_pool.rb

```diff
@@ -88,9 +88,8 @@ def buy_shares(entity, shares, exchange: nil, exchange_price: nil, swap: nil, al
       if price.zero?
         transfer_shares(bundle, entity)
       else
-        receiver = if corporation.capitalization == :incremental && bundle.owner.corporation? || bundle.owner.player?
-                     bundle.owner
-                   elsif corporation.capitalization == :escrow && bundle.owner.corporation?
+        receiver = if %i[escrow incremental].include?(corporation.capitalization) && bundle.owner.corporation? ||
+                       bundle.owner.player?
                      # When another game with escrow capitalization is implemented put this into @game
```

Comment:
Don't know if this comment still makes sense here now that the `escrow` if block has been combined with the `incremental` one

Reply:
(also I don't even know what `escrow` capitalization means 😝 )

Reply:
@ventusignis Please take a look, if and where the comment would still make sense.

Reply:
There's another similar line elsewhere in this file at a point where there is actually different logic for escrow, I think the comment here can be removed without issue.

Reply:
I went ahead and removed the comment. 🪓